_includes/promos/10tvpromo.html

```diff
@@ -0,0 +1,106 @@
+<div class="jumbotron background-white">
+    <div class="row">
+        <div class="col-md-9">
+            <h1 style="display:inline; font-family: Roboto;
+            font-size: 54px;
+            font-weight: 500;
+            letter-spacing: 0.43px;
+            line-height: 71px;">Thank you 10tv</h1>
+            <h4><br/>10tv has partnered with Can't Stop Columbus and has graciously<br />volunteered to give free advertizing space on their <a href="https://www.10tv.com/">website</a>.</h4>
+            <h4><br/>Offer valid until May 1st, 2020<br />To submit your business, fill out the form below.</h4>
+        </div>
+        <div class="col-md-3">  
+            <br><img style="max-width: 100%;
+            height: auto;" src="assets/img/UX/logos/10tv.jpg">
+        </div>
+    </div>
+    <br />
+    <br />
+    <br />
```

Comment:
Why the `<br />`? Why not add some padding or margin?

Reply:
Good point. I can get that fixed!